pact.cabal

```diff
@@ -220,6 +221,7 @@ library
     , deriving-compat >=0.5.1
     , direct-sqlite >=2.3.27
     , directory >=1.2.6.2
+    , data-dword
```

Comment:
i recommend using [wide-word](https://hackage.haskell.org/package/wide-word), which has `binary` instances available for all the stuff you're needing to do (and is done more optimally).

Reply:
Awesome, thx!

Reply:
We checked and this instance is the same on all machines, so it works fine for us.